flink-runtime/src/main/java/org/apache/flink/runtime/util/bash/BashJavaUtils.java

```diff
@@ -92,9 +102,42 @@ public static void main(String[] args) throws FlinkException {
 		JobManagerProcessSpec jobManagerProcessSpec = JobManagerProcessUtils.processSpecFromConfigWithNewOptionToInterpretLegacyHeap(
 			configuration,
 			JobManagerOptions.JVM_HEAP_MEMORY);
+
+		logMasterConfiguration(jobManagerProcessSpec);
+
 		return Collections.singletonList(ProcessMemoryUtils.generateJvmParametersStr(jobManagerProcessSpec));
 	}
 
+	private static void logMasterConfiguration(JobManagerProcessSpec spec) {
+		JobManagerFlinkMemory flinkMemory = spec.getFlinkMemory();
+		LOG.info("Final Master Memory configuration:");
+		LOG.info("  Total Process Memory: {}", spec.getTotalProcessMemorySize().toHumanReadableString());
+		LOG.info("    Total Flink Memory: {}", flinkMemory.getTotalFlinkMemorySize().toHumanReadableString());
+		LOG.info("      Heap:             {}", flinkMemory.getJvmHeapMemorySize().toHumanReadableString());
```

Comment:
```suggestion
		LOG.info("      JVM Heap:         {}", flinkMemory.getJvmHeapMemorySize().toHumanReadableString());
```

Reply:
I think we do not use `Total` for heap in docs, mostly `JVM heap`

Reply:
Although, I am also ok with `Total`, I would add `JVM` at least to align terminology more with docs.